lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/EagerFallback.cpp

```diff
@@ -0,0 +1,195 @@
+#include "lazy_tensor_core/csrc/ts_backend/EagerFallback.h"
+
+#include <ATen/Functions.h>
+#include <ATen/core/boxing/KernelFunction.h>
+#include <ATen/native/CPUFallback.h>
+#include <torch/library.h>
+
+namespace torch_lazy_tensors {
+namespace {
+
+std::vector<at::Tensor> _to_device(at::TensorList tensors,
+                                   c10::Backend backend) {
+  c10::DeviceType device_type = c10::backendToDeviceType(backend);
+  std::vector<at::Tensor> eager_tensors;
+  for (const auto& t : tensors) {
+    at::Tensor eager_tensor = t.to(t.options().device(device_type),
+                                   /*non_blocking*/ false, /*copy*/ false);
+    eager_tensors.push_back(eager_tensor);
+  }
+  return eager_tensors;
+}
+
+std::vector<at::Tensor> _to_eager(at::TensorList tensors,
```

Comment:
i'm not sure why _to_eager_ has a special case for eager cpu, but not eager cuda.  is eager cuda handled by _to_device, or just not handled yet?

Reply:
The context that I have is that xla requires special handling when converting tensors to cpu. Which we abstract over with the `at::_to_cpu()` op (it's implemented in core with a for-loop over the tensor args, but xla has special handling to move them all simultaneously).

I'm wondering if other vendors aside from XLA will want similar functionality available when falling back to cuda though

Reply:
What @bdhirsh said is correct, I'm special casing `at::_to_cpu` for XLA needs. There's no `at::_to_cuda` yet, so I'm letting the other cases fall through.